libs/template/experimental/virtual-scrolling/src/lib/virtual-scroll-viewport.component.ts

```diff
@@ -142,7 +147,7 @@ export class RxVirtualScrollViewportComponent
    * in sync with the DOM when the next `renderCallback` emitted an event.
    */
   @Output()
-  readonly viewRange = this.scrollStrategy.renderedRange$;
+  readonly viewRange = this.scrollStrategy?.renderedRange$;
```

Comment:
I think we can remove the optional chaining here, `scrollStrategy` should always be defined.

Reply:
done